extensions.json

```
@@ -940,6 +940,11 @@
       "id": "Nightrains.robloxlsp",
       "repository": "https://github.com/NightrainsRbx/RobloxLsp"
     },
+    {
+      "id": "nonoroazoro.syncing",
+      "repository": "https://github.com/nonoroazoro/vscode-syncing",
+      "version": "3.1.0"
```

Since we're pinning a specific version, we should probably publish from the corresponding release tag (instead of the repository's default branch, which may contain WIP changes at the time of publishing):

```suggestion
      "version": "3.1.0",
      "checkout": "v3.1.0"
```